controllers/api/index.js

```diff
@@ -1,5 +1,6 @@
-const router = require('express').Router();
+const express = require('express');
```

Comment:
Lines 1 and2 set up links express server. An express server is already set up with links to its routes in server.js. These lines can be removed.